src/cache/asset.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'file_path'
+require 'data'
+
+module Metalware
+  module Cache
+    class Asset
+      def data
+        @assets ||= begin
```

Comment:
As the is the cache for the `data` method, the instance variable should also be called data (as opposed to `asset`)